ruspy/simulation/simulation.py

```diff
@@ -6,14 +6,15 @@
 """
 import numpy as np
 import pandas as pd
-import scipy.stats as stats
-from ruspy.simulation.simulation_auxiliary import simulate_strategy
-from ruspy.estimation.estimation_cost_parameters import lin_cost
-from ruspy.estimation.estimation_cost_parameters import cost_func
-from ruspy.simulation.simulation_auxiliary import simulate_strategy_loop_known
+from ruspy.simulation.simulation_auxiliary import (
+    simulate_strategy,
+    get_unobs,
+    get_increments,
+)
+from ruspy.estimation.estimation_cost_parameters import lin_cost, cost_func
 
 
-def simulate(init_dict, ev_known=None, shock=None):
+def simulate(init_dict, ev_known, real_trans_mat, shock=None):
```

Comment:
There are not different concepts of transition matrix in this function, so `real` is not needed.